shared/js/background/utils.es6.js

```diff
@@ -317,39 +317,6 @@ function isDomainCookieExcluded (domain) {
     return false
 }
 
-/**
- * Convert an image file to a base64 data:image file,
- * for use in injections where the extension URL may not be
- * accessible
- */
-export async function imgToData (imagePath) {
```

Comment:
I noticed [imgToData was added with the original Click to Load commit](https://github.com/duckduckgo/duckduckgo-privacy-extension/commit/624274921b521c219183f23ad0482780e37a4fc6#diff-fa4895e8a5f9f011f418b7e04b16ec46ac30764db15368ae60a2d6820962df9b), and I can't see where else it's used.